From 90d07b7ef09929a0081d0dca6d87be7f8858f379 Mon Sep 17 00:00:00 2001
Message-Id: <90d07b7ef09929a0081d0dca6d87be7f8858f379.1414996573.git.chang-joon.lee@intel.com>
In-Reply-To: <de6d11b8587be186d089e692fc410008d4d87034.1414996573.git.chang-joon.lee@intel.com>
References: <de6d11b8587be186d089e692fc410008d4d87034.1414996573.git.chang-joon.lee@intel.com>
From: Rafael Barbalho <rafael.barbalho@intel.com>
Date: Fri, 31 Oct 2014 11:51:56 +0000
Subject: [PATCH 08/10] FOR_UPSTREAM [VPG]: drm/i915: Remove unused variable
 in vlv_configure_dsi_pll

The following commit didn't remove a variable:

commit 39e6b356786af37788941e9eaa4329ad42a923f4
Author: Yogesh Mohan Marimuthu <yogesh.mohan.marimuthu@intel.com>
Date:   Thu Oct 16 18:59:37 2014 +0530

    FOR_UPSTREAM [VPG]: drm/i915: program DSI PLL from intel_dsi pixel clk

For: GMINL-1778
Change-Id: Idada3b3c82b4eaae17e4da5e0c9ca839b181d79e
Signed-off-by: Rafael Barbalho <rafael.barbalho@intel.com>
---
 drivers/gpu/drm/i915/intel_dsi_pll.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_dsi_pll.c b/drivers/gpu/drm/i915/intel_dsi_pll.c
index c398c2e..85cbd21 100644
--- a/drivers/gpu/drm/i915/intel_dsi_pll.c
+++ b/drivers/gpu/drm/i915/intel_dsi_pll.c
@@ -262,7 +262,6 @@ static void vlv_configure_dsi_pll(struct intel_encoder *encoder)
 {
 	struct drm_i915_private *dev_priv = encoder->base.dev->dev_private;
 	struct intel_crtc *intel_crtc = to_intel_crtc(encoder->base.crtc);
-	const struct drm_display_mode *mode = &intel_crtc->config.adjusted_mode;
 	struct intel_dsi *intel_dsi = enc_to_intel_dsi(&encoder->base);
 	enum pipe pipe = intel_crtc->pipe;
 	int ret;
-- 
1.7.9.5

